content/sensu-go/5.12/installation/auth.md

```diff
@@ -1094,6 +1099,412 @@ example      | {{< highlight shell >}}"name": "activedirectory"{{< /highlight >}
 
 See the [LDAP troubleshooting](#ldap-troubleshooting) section.
 
+## OIDC authentication
+
+The Sensu offers license-activated support for OIDC driver for using the OpenID Connect 1.0 protocol (OIDC) on top of the OAuth 2.0 protocol for RBAC authentication.
+
+### OIDC configuration examples
```

Comment:
Need to update these examples with the `sensu-go` "format"; here is the one we were using to test during QA:

```
$ sensuctl auth info oidc --format yaml
type: OIDC
api_version: authentication/v2
metadata:
  name: oidc
spec:
  additional_scopes:
  - email
  - groups
  client_id: 0oa13ry4ypeDDBpxF357
  client_secret: DlArQRfND4BKBUyO0mE-TL2PWOVwyGjIO1fdk9gX
  groups_claim: groups
  groups_prefix: 'oidc:'
  redirect_uri: https://34.215.21.225:8080/api/enterprise/authentication/v2/oidc/callback
  server: https://sensu-cyril.okta.com
  username_claim: email
  username_prefix: 'oidc:'
```

Also, the struct describing the config is here if that helps:
https://github.com/sensu/sensu-enterprise-go/blob/master/api/authentication/v2/oidc.go#L19-L52